Makefile

```diff
@@ -269,3 +276,42 @@ glance_deploy_cleanup: ## cleans up the service instance, Does not affect the op
 	$(eval $(call vars,$@,glance))
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${OPERATOR_BASE_DIR}/glance-operator ${DEPLOY_DIR}
+
+##@ NEUTRON
+.PHONY: neutron_prep
+neutron_prep: export IMAGE=${NEUTRON_IMG}
+neutron_prep: ## creates the files to install the operator using olm
+	$(eval $(call vars,$@,neutron))
+	bash scripts/gen-olm.sh
+
+.PHONY: neutron
```

Comment:
neutron should be added to all, deploy and deploy_cleanup at the top

Reply:
Added to all, cleanup and deploy_cleanup